plugins/jira/skills/status-analysis/scripts/summarize_issue.py

```
@@ -0,0 +1,316 @@
+                    labeled.append(arg)
+                else:
+                    key = d["issue"]["key"]
+                    summary = d["issue"]["summary"]
+                    unlabeled.append({"arg": arg, "key": key, "summary": summary})
+            except (FileNotFoundError, json.JSONDecodeError):
+                labeled.append(arg)
+        args = labeled
+        if unlabeled:
+            print(f"=== MISSING LABEL '{label_filter}' ({len(unlabeled)} issues) ===")
+            print(f"These issues appeared as descendants but do not carry the '{label_filter}' label.")
+            print(f"Confirm with the user whether to include them in the report.\n")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)